qualtran/bloqs/block_encoding.py

```diff
@@ -182,69 +191,247 @@ def pretty_name(self) -> str:
         return 'Prep'
 
 
+class BlockEncoding(Bloq):
+    r"""Abstract interface for an arbitrary block encoding.
+
+    In general, given an $s$-qubit operator $H$ then the $(s+a)$-qubit unitary $U$ is
+    a $(\alpha, a, \epsilon)$-block encoding of $H$ if it satisfies:
+
+    $$
+        \lVert H - \alpha (\langle G|_a\otimes I_s U |G\rangle_a \otimes I_s) \rVert
+        \le \epsilon,
+    $$
+
+    where $a$ is an ancilla register and $s$ is the system register, $U$ is a unitary sometimes
+    called a signal oracle, $\alpha \ge \lVert H\rVert$ (where $\lVert \cdot \rVert$
```

Comment:
nit: say what alpha is rather than just its magnitude. e.g. below you call it "normalization constant alpha"

Reply:
Done.